src/administration/install_docker.md

```diff
@@ -1,78 +1,118 @@
+
+Some common choices include webservers such as [NGINX](https://www.nginx.com/), or a dedicated proxy such as [Traefik](https://traefik.io/). Configuration is _not_ limited to these two options, however these are the most common and example configurations are provided for them.
+
+The Reverse proxy may be part of the same Docker Compose file as the Lemmy server and UI, or it may be a separate service. In a production configuration it is recommended that the reverse proxy be a separate service, as this allows for greater flexibility in scaling and configuration.
+
+## Deployment
+
+Create a location to store the Lemmy docker environment; set any required permissions.
 
 ```bash
 # Create a folder for the lemmy files. the location doesnt matter, you can put this anywhere you want
-mkdir lemmy
-cd lemmy
+mkdir -p /data/lemmy/{volumes,config}
+mkdir -p /data/lemmy/volumes/{pictrs,postgres}
+chown 991:991 /data/lemmy/volumes/pictrs
+cd /data/lemmy
 ```
 
-## Download default config files
+### Download an appropriate Docker template
+
+There are many ways that the Lemmy docker environment can be configured. The following sections describe some of the more common configurations.
 
-These files contain `{{ }}` braces for variables, such as passwords and your domain.
+#### NGINX
 
-Edit them before starting up lemmy for the first time.
+Appropriate templates for nginx are available in the [lemmy-ansible](https://github.com/LemmyNet/lemmy-ansible) repository. While these are designed to be deployed via Ansible automation, the templates can be trivially hand-edited.
 
-The images will likely be: [dessalines/lemmy:VERSION](https://hub.docker.com/r/dessalines/lemmy) and [dessalines/lemmy-ui:VERSION](https://hub.docker.com/r/dessalines/lemmy-ui)
 
 ```bash
-wget https://raw.githubusercontent.com/LemmyNet/lemmy-ansible/main/templates/docker-compose.yml
-wget https://raw.githubusercontent.com/LemmyNet/lemmy-ansible/main/examples/config.hjson -O lemmy.hjson
-wget https://raw.githubusercontent.com/LemmyNet/lemmy-ansible/main/templates/nginx_internal.conf
+curl -O https://raw.githubusercontent.com/LemmyNet/lemmy-ansible/main/templates/docker-compose.yml
+pushd config
+curl -O https://raw.githubusercontent.com/LemmyNet/lemmy-ansible/main/examples/config.hjson
+curl -O https://raw.githubusercontent.com/LemmyNet/lemmy-ansible/main/templates/nginx_internal.conf
+popd
 ```
 
-If you'd like further customization, have a look at the [config file](configuration.md) named `lemmy.hjson`, and adjust it accordingly.
+These templates contain `{{ }}` braces for variables, such as passwords and the domain name; edit them before starting up Lemmy for the first time.
 
-## Database tweaks
+#### Traefik
```

Comment:
Remove this, we'll only support nginx. Traefik or the dozens of other potential servers should have instructions elsewhere.

Reply:
> Remove this, we'll only support nginx. Traefik or the dozens of other potential servers should have instructions elsewhere.

I'd prefer not to do this. The entire rationale for revisiting these docs was to document Traefik as a load balancer / reverse proxy for Lemmy as it's been noted in several places as being ~~lacking~~ nonexistant; the docker documentation getting a touch-up is just a happy side-effect.